views/internal.go

```diff
@@ -36,31 +38,31 @@ func (v *Views) InternalFunc(c echo.Context) error {
 	if err != nil {
 		log.Println(err)
 		if !v.conf.Debug {
-			return v.errorHandle(c, err)
+			return echo.NewHTTPError(http.StatusInternalServerError, fmt.Errorf("failed to get number of users for interal: %w", err))
```

Comment:
```suggestion
			return fmt.Errorf("failed to get number of users for interal: %w", err)
```

(and dto for the others)

Echo will (by default) convert anything that isn't an `echo.HTTPError` into a 500

Reply:
Not anymore, will have custom errors